params/state_upgrade.go

```diff
@@ -42,7 +42,27 @@ func (s *StateUpgrade) Equal(other *StateUpgrade) bool {
 type stateUpgradeAccountMarshaling struct {
 	Code          hexutil.Bytes
 	Storage       map[storageJSON]storageJSON
-	BalanceChange *math.HexOrDecimal256
+	BalanceChange *SignedHexOrDecimal256
+}
+
+type SignedHexOrDecimal256 math.HexOrDecimal256
```

Comment:
Alternatively we can split this to `AddBalance` and `SubBalance`, but I think this is fine.